web/themes/custom/usagov/templates/block--usagov-local-tasks.html.twig

```diff
@@ -0,0 +1,65 @@
+{# {% set site_url = url("<current>") %}
+{% if 'states' in site_url['#markup']|render|render %} #}
+<p class="usa-intro">
```

Comment:
I would remove this from the twig file and put it in the "Intro" on the content node that winds up hosting this. That will make it easy for content editors to tweak the text in the future if they want to.